src/main/java/hudson/plugins/dimensionsscm/DimensionsAPI.java

```diff
@@ -22,6 +25,7 @@
 import hudson.model.Node;
 import hudson.model.Run;
 import hudson.util.Secret;
+
```

Comment:
Standard import order for Jenkins plugin is just alphabetical, with no blank lines in the import section, only a single blank line before and after the imports.